### PR TITLE
Added app insights

### PR DESCRIPTION
**Pull request by amrutha95**
As part of this [issue](https://github.com/pwa-builder/PWABuilder/issues/3891)

## PR Type

Feature 

## Describe the current behavior?
Adds the new analytics to android platform

## Describe the new behavior?
Data is sent to app insights

## PR Checklist

- [ ] Test: run `npm run test` and ensure that all tests pass
- [ ] Target master branch (or an appropriate release branch if appropriate for a bug fix)
- [ ] Ensure that your contribution follows [standard accessibility guidelines](https://docs.microsoft.com/en-us/microsoft-edge/accessibility/design). Use tools like https://webhint.io/ to validate your changes.

### CODE REVIEW

File: .gitignore [hunk 1]
```diff
@@ -11,6 +11,9 @@ pids
 *.seed
 *.pid.lock
 
+#remove js files
```
Comment (nmetulev) [inline .gitignore:14]: I see changes in js files here, should I ignore them? Should we remove the js files that are already included?

**Comment by amrutha95** [comment]
Removed js files but making a few more changes. Do not merge yet!

File: .gitignore (hunk 1 above)
Comment (amrutha95) [reply .gitignore:14]: I've removed the js files

**Comment by amrutha95** [comment]
Ready to merge! You can test this on the preview environment: pwabuilder-cloudapk-pre.azurewebsites.net

**Review by nmetulev — APPROVED** [review]

**Pull request merged**